util.bzl

```diff
@@ -20,7 +20,7 @@ def hazel_ghclibs(version, shaParser, shaLibrary):
         (
             "ghc-lib",
             {
-                "url": "https://digitalassetsdk.bintray.com/ghc-lib/ghc-lib-" + version + ".tar.gz",
+                "url": "https://purelyfunctional.org/downloads/ghc-lib-" + version + ".tar.gz",
```

Comment:
What's this about?

Reply:
Only temporary to test CI until I make a proper `ghc-lib` release.